xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1468,7 +1468,7 @@ activitystream.event.deleteComment.rss.body=A comment has been deleted from the
 ### Deleting a page
 core.delete=Delete
 core.delete.title=Delete {0}
-core.delete.backlinksWarning=The following pages contain links to the current page:{0}After deleting this page, those links will point to an empty page.
```

Comment:
Has this been deprecated?

Reply:
No, I forgot this is the practice for deleted keys. Thanks!